pkg/services/authz/proto/v1/extention.proto

```diff
@@ -86,3 +88,26 @@ message WriteRequest {
 }
 
 message WriteResponse {}
+
+message BatchCheckRequest {
+  string subject = 1;
+  string namespace = 2;
+  repeated BatchCheckItem items = 3;
+}
+
+message BatchCheckItem {
+  string verb = 1;
+  string group = 2;
+  string resource = 3;
+  string name = 4;
+  string subresource = 5;
+  string folder = 6;
+}
+
+message BatchCheckResponse {
+  map<string, BatchCheckGroup> groups = 1;
```

Comment:
You need to update the name here to `BatchCheckGroupResource` and re-generate protobufs